data/scenarios/Challenges/_blender/solution.sw

```diff
@@ -0,0 +1,64 @@
+def doN = \n. \f. if (n > 0) {f; doN (n - 1) f} {}; end;
+
+def slowM = move; wait 2; end;
+
+def navigatePath =
+    doN 9 move;
+    turn left;
+    wait 13; // Get this timing right
+    doN 12 slowM;
+    turn right;
+    doN 8 move;
+    turn right;
+
+    wait 0; // Get this timing right
+    doN 12 slowM;
+    turn left;
+    doN 8 move;
+
+    turn left;
+    wait 0; // Get this timing right
+    doN 12 slowM;
+    turn left;
+    move;
+
+    k <- grab;
+    equip k;
+
+    doN 5 move;
+    drill forward;
+
+    doN 2 move;
+    turn left;
+
+    wait 0; // Get this timing right
+    doN 4 move;
+    turn right;
+    drill forward;
+    doN 2 move;
+
+    // Close the door behind us!
+    drill back;
```

Comment:
This doesn't seem to be required --- the patrolling robots don't seem to go through unlocked doors even if it's the direction they would normally turn.  When I was solving the challenge I didn't think to lock the door behind me and I didn't have an issue.  I like this though --- I think we should fix the patrolling robots so that they *do* go through unlocked doors, it makes it more fun/interesting.